### DISCUSSION TITLE
Postgres extract()

### DISCUSSION THREAD

**Discussion by Perry-Olsson** [prisma/prisma#5719] [Q&A] [answered]
I'm wondering if there is any way (other than using a raw query) to make use of the extract function in Postgres with the PrismaClient. 

The raw query I'm using right now: 

SELECT * FROM "Appointment" WHERE extract(month from timestamp) = 2;

**Comment by ryands17** [comment, marked as answer]
@Perry-Olsson 👋 

Currently this is only possible via a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) but I would suggest creating a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) that allows raw function access inside Prisma APi's.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
